18F/identity-idp
# Fix per-component assets for component previews

## 🛠 Summary of changes

Fixes an issue where component previews were not loading assets correctly for per-component assets.

## 📜 Testing Plan

1. Go to http://localhost:3000/components/inspect/clipboard_button/preview
2. Click the button
3. Observe that the tooltip is shown
   - This was previously broken in two ways: The stylesheet responsible for the tooltip styles was not being loaded (404ing), and neither was the JavaScript which initialized the component.
